collectors/rust/test-coverage.sh

```
@@ -0,0 +1,57 @@
+
+# Check for llvm-cov output
+if [[ "$CMD_STR" == *"llvm-cov"* ]]; then
+    tool="cargo-llvm-cov"
+    # llvm-cov can output JSON summary
+    if [[ -f "coverage-summary.json" ]]; then
+        coverage_pct=$(jq -r '.data[0].totals.lines.percent // empty' coverage-summary.json 2>/dev/null || true)
+    fi
+    # Check for lcov.info
+    if [[ -z "$coverage_pct" ]] && [[ -f "lcov.info" ]]; then
+        # Parse lcov format: sum of LH (lines hit) / LF (lines found)
+        lh=$(grep -c '^LH:' lcov.info 2>/dev/null || echo "0")
```

Shellcheck (0.11.0)

[warning] lh appears unused. Verify use (or export if used externally).

(SC2034)